.circleci/config.yml

```diff
@@ -25,7 +27,8 @@ common-steps:
       name: Install Debian packaging dependencies and download wheels
       command: |
         mkdir ~/packaging && cd ~/packaging
-        git config --global --unset url.ssh://git@github.com.insteadof
+        # local builds may not have an SSH url
+        git config --global --unset url.ssh://git@github.com.insteadof || true
```

Comment:
This line can be removed entirely, it was only needed because the circleci images were...weird.

Reply:
Uhhh, egg on my face. I guess this is still required, not sure why but it is.